toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -41,7 +41,8 @@ const Popup = class {
 		}
 		const pos = this._calcPositioning();
 		this._content.style.top = this._px(pos.top);
-		this._content.style.transform = `translateX(${pos.left}px)`;
+		this._content.style.left = this._px(pos.left);
+		this._content.style.right = this._px(5);
```

Comment:
Is 5 a magic number? Is it always the case that this is 5?

Reply:
5 is indeed the magic number, De La Soul were wrong all along.

I'm going to make 5 the initial value and use this as a variable that can be passed into both the right and left by default. If it needs to change the code will do that. It's basically passed a left/right value to stretch the popup.